cli/src/pixl_cli/_database.py

```diff
@@ -14,8 +14,9 @@
 
 """Interaction with the PIXL database."""
 
-from core.db.models import Extract, Image
-from core.patient_queue.message import Message
+from core.db.models import Extract, Image  # type: ignore [import-untyped]
+from core.patient_queue.message import Message  # type: ignore [import-untyped]
+from slugify import slugify  # type: ignore [import-untyped]
```

Comment:
how come these are popping up just now?

Reply:
🤷‍♂️ tbf they typically pass pre-commit but keep popping up locally for me in my editor

Reply:
`Skipping analyzing "pytest_pixl.dicom": module is installed, but missing library stubs or py.typed marker`
Might just be a configuration thing maybe